### PR TITLE
Display fiat amount and currency on same line

### PR DESCRIPTION
**Pull request by josepfo**

### CODE REVIEW

**Comment by josepfo** [comment]
[Ensure fiat amount and currency display on same line](https://trello.com/c/4wOO84IY/617-ensure-fiat-amount-and-currency-display-on-same-line)

File: new-lamassu-admin/src/pages/Transactions/Transactions.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ const Transactions = () => {
     },
     {
       header: 'Cash',
-      width: 104,
+      width: 110,
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Transactions/Transactions.js]: How many more chars we can fit in this amount of pixels?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Transactions/Transactions.js (hunk 1 above)
Comment (josepfo) [reply new-lamassu-admin/src/pages/Transactions/Transactions.js]: If the amount reaches 4 digits it goes to a new line. Should I increase the width even more?

Comment (RafaelTaranto) [reply new-lamassu-admin/src/pages/Transactions/Transactions.js]: Yeah, put date and time together and get as much as you can here and in crypto

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**